api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2304,7 +2304,7 @@ private void enableEIDForm() {
                 Privileges.TASK_EMR_ENTER_HIV_CONSULT_NOTE.privilege(),
                 and(sessionLocationHasTag("HIV Consult Location"),
                         visitDoesNotHaveEncounterOfType(PihEmrConfigConstants.ENCOUNTERTYPE_EID_FOLLOWUP_UUID),
-                        and(patientYoungerThan(2)),
+                        and(patientYoungerThanMonths(18)),
```

Comment:
This should either be patientYoungerThanMonths(19) or patientYoungerThanOrEqualToMonths(18), depending on how you decide to change things below.